api/auth/center.go

```diff
@@ -314,7 +315,7 @@ func cloneRequest(r *http.Request, authHeader *authHeader) *http.Request {
 
 func (c *center) checkSign(authHeader *authHeader, box *accessbox.Box, request *http.Request, signatureDateTime time.Time) error {
 	awsCreds := credentials.NewStaticCredentials(authHeader.AccessKeyID, box.Gate.AccessKey, "")
-	signer := v4.NewSigner(awsCreds)
+	signer := v4amz.NewSigner(awsCreds)
```

Comment:
Why it was added initially?

Reply:
Our implementation?

Reply:
Yes.

Reply:
https://github.com/nspcc-dev/neofs-s3-gw/issues/528

Reply:
Maybe https://github.com/nspcc-dev/neofs-s3-gw/issues/1028 can help here.